streaming/src/main/resources/org/apache/spark/streaming/ui/static/streaming-page.js

```diff
@@ -169,7 +169,7 @@ function drawTimeline(id, data, minX, maxX, minY, maxY, unitY, batchInterval) {
             .style("cursor", "pointer")
             .attr("cx", function(d) { return x(d.x); })
             .attr("cy", function(d) { return y(d.y); })
-            .attr("r", function(d) { return isFailedBatch(d.x) ? "2" : "0";})
+            .attr("r", function(d) { return isFailedBatch(d.x) ? "2" : "3";})
```

Comment:
This changes the radius of a blue circle (which represents a successful batch) in the svg.

Prior to this patch, the radius was 0 thus the blue circle could not be hovered and tooltip would not show. This patch alters the radius to 3 for each blue circle.